pkg/network/kubemacpool.go

```diff
@@ -91,9 +106,9 @@ func renderKubeMacPool(conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir str
 	data.Data["RangeStart"] = conf.KubeMacPool.RangeStart
 	data.Data["RangeEnd"] = conf.KubeMacPool.RangeEnd
 
-	objs, err := render.RenderDir(filepath.Join(manifestDir, "kubemacpool"), &data)
```

Comment:
this change introduced a bug, it has to remain lowercase

Reply:
Good catch!